nginx-directory/configfile.libsonnet

```diff
@@ -11,8 +11,10 @@ function(services) {
         return 302 %(url)s;
       ||| % service
       else |||
-        proxy_pass      %(url)s$2$is_args$args;
-        proxy_set_header    Host $host;
+        #automagically replaces non-regex location identifier with given proxy_pass url.
```

Comment:
nit: My eyes dislike comments that don't start with a space but it may just be me.

Reply:
fixed.

Reply:
Thanks!